openxcframework/VehicleManager.swift

```diff
@@ -1438,10 +1438,10 @@ open class VehicleManager: NSObject {
   // TODO: ToDo - Uncomment the code when when there will be a server URL and test the code
 
   //Send data using trace URL
-  @objc public func sendTraceURLData(urlName:String,rspdict:NSMutableDictionary,isdrrsp:Bool) {
+    @objc public func sendTraceURLData(urlName:String,rspdict:NSMutableDictionary,isdrrsp:Bool) {
 
     //declare parameter as a dictionary which contains string as key and value combination. considering inputs are valid
-    //"http://localhost:8080/print"
+    //var base64String = "my fancy string".data(using: .utf8, allowLossyConversion: false)?.base64EncodedString()
```

Comment:
If this code is being used, it should be uncommented. Otherwise it should be removed.

Reply:
it is removed in latest one